README.md

```diff
@@ -57,35 +56,50 @@ six total versions detailed below:
 
 The scaling repo offers two types of Orchestrator deployement: Standard and Colocated.
 
+> Note that the Orchestrator is the name in SmartSim for a Redis or KeyDB database with a RedisAI module built into it with the ML runtimes.
+
 1. `Colocated (non-Clustered Deployement)`
-  : When running a Colocated test, your database will be deployed on the same node as your application.
+  : A colocated Orchestrator is a special type of Orchestrator that is deployed on the same compute hosts as the application. 
+  This is particularly important for GPU-intensive workloads which require frequent communication with the database. You can specify the number of nodes via the `client_nodes` flag. 
 
 2. `Standard (Clustered Deployement)`
   : When running with Standard deployement, your database will be deployed on different compute nodes
-than your application. You will notice that all Standard tests share a `db_nodes` flag. By setting the flag to `db_nodes=[4,8]` - you are telling the program to split up your database to four shards on the first permutation, then eight shards on the second permutation.
+than your application. You will notice that all Standard scaling tests share a `db_nodes` flag. By setting the flag to `db_nodes=[4,8]` - you are telling the program to split up your database to four shards on the first permutation, then eight shards on the second permutation. Each shard of the database will communicate with each application node. You can specify the number of application nodes via the `client_nodes` flag in each scaling test.
 
```

Comment:
I think you should also describe here that we have scaling testers for both C++ and Fortran